Makefile

```diff
@@ -176,10 +177,7 @@ $(generated_code_deep_copy_helper): $(deep_copy_helper_input) .license-header ##
 	./tools/update-codegen.sh
 
 $(generated_code_aws_sdk_mocks): $(call godeps,pkg/eks/mocks/mocks.go)
-	mkdir -p vendor/github.com/aws/
```

Comment:
Remove this hack for mockery.

Golang 1.14 is not auto detecting vendor mode (e.g. if vendor folder is in root project direction, compiler will just change to `vendor-mod` instead of `mod`), so this changes are required :)